docs/book/v6/extended-features/core-and-app.md

```diff
@@ -0,0 +1,34 @@
+# Core and App code structure
+
+In the 6.0 version, the project is split into two main parts: **App** and **Core**.
+
+The purpose is to reach a headless CMS format for easier scalability.
+Headless CMS is a back-end-only content management system that acts primarily as a content repository.
+Compared to traditional CMS platforms (e.g WordPress) that tightly couple the front end and back end, a headless CMS decouples the content management from the presentation layer.
+The content is delivered through APIs allowing any front-end to fetch and display it, making front-end and back-end development easier to work in parallel.
+
+## What is "App" and what is "Core"?
+
+### Core
+
+The **Core** like the backbone of the application.
+It's where the core logic lives.
```

Comment:
It's where the core logic lives.
to
It contains the core logic.
maybe?
It contains the core logic, the lowest-level features.